### PR TITLE
adds a bone sword and a goliath shield (Port)

### PR DESCRIPTION
**Pull request by boodaliboo**
Original Pr:https://github.com/BeeStation/BeeStation-Hornet/pull/1802 
this pr adds a new bone sword and a goliath shield.
this is good as it expands on ashwalker content and gives them more to do beside killing
#### Changelog

:cl:  
rscadd: Added bone sword and goliath shield
imageadd: added sword and shield sprite
/:cl:

### CODE REVIEW

File: code/game/objects/items/shields.dm [hunk 1]
```diff
@@ -118,6 +118,25 @@
...
+/obj/item/shield/riot/goliath
+	name = "Goliath shield"
+	desc = "A shield made from interwoven plates of goliath hide."
+	icon_state = "goliath_shield"
+	item_state = "goliath_shield"
+	lefthand_file = 'icons/mob/inhands/equipment/shields_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/shields_righthand.dmi'
+	materials = list ()
+	transparent = FALSE
+	block_power = 25
```
Comment (jcatjmeow) [inline code/game/objects/items/shields.dm]:
```suggestion
	block_chance = 25
```

**Review by jcatjmeow — APPROVED** [review]
Looks fine

**Comment by Takahiru** [comment]
it better not have a missing sprite

**Review by Partheo — APPROVED** [review]

**Comment by Marmio64** [comment]
that spritemaker is pretty damn good

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**